src/virmet/wolfpack.py

```diff
@@ -23,13 +23,17 @@
                   '/data/virmet_databases/bacteria/bwa/bact1',
                   '/data/virmet_databases/bacteria/bwa/bact2',
                   '/data/virmet_databases/bacteria/bwa/bact3',
+                  '/data/virmet_databases/bacteria/bwa/bact4',
+                  '/data/virmet_databases/bacteria/bwa/bact5',
                   '/data/virmet_databases/fungi/bwa/fungi1',
                   '/data/virmet_databases/bovine/bwa/bt_ref']
 ref_map = {
     'humanGRCh38': '/data/virmet_databases/human/fasta/GRCh38.fasta.gz',
     'bact1': '/data/virmet_databases/bacteria/fasta/bact1.fasta.gz',
     'bact2': '/data/virmet_databases/bacteria/fasta/bact2.fasta.gz',
     'bact3': '/data/virmet_databases/bacteria/fasta/bact3.fasta.gz',
+    'bact4': '/data/virmet_databases/bacteria/fasta/bact4.fasta.gz',
+    'bact5': '/data/virmet_databases/bacteria/fasta/bact5.fasta.gz',
```

Comment:
Since you have a `N_FILES_BACT` now, you could compile this dictionary and the array before programmatically.